src/ledger/LedgerManagerImpl.cpp

```diff
@@ -560,6 +593,118 @@ LedgerManagerImpl::getSorobanMetrics()
     return mSorobanMetrics;
 }
 
+::rust::Box<rust_bridge::SorobanModuleCache>
+LedgerManagerImpl::getModuleCache()
+{
```

Comment:
Nit: I'd be interested in adding tracy zones to these functions. Especially as the ledger grows, might be interesting to see how we block on these calls.

Reply:
yep yep yep